spec/embedded_sass.proto

```diff
@@ -1066,10 +1068,12 @@ enum CalculationOperator {
   DIVIDE = 3;
 }
 
-// The built-in Node Package Importer, which is a Package Importer that resolves
-//using the standards and conventions of the Node ecosystem. It enables a `pkg:`
-// URL scheme for usage with `@use` that directs an implementation to resolve a
-// URL within a dependency. It must be instantiated with an`entry_point_url`.
+// The built-in Node.js Package Importer, which is a Package Importer that
+// resolves using the standards and conventions of the Node.js ecosystem. It
+// enables a `pkg:` URL scheme for usage with `@use` that directs an
+// implementation to resolve a URL within a dependency. It must be instantiated
+// with an`entry_point_url`, which is the path to a file within the NPM package
+// that contains the NPM packages that will be imported using the `pkg:` scheme.
```

Comment:
Since this is effectively spec text, rather than sort of vaguely describing `entry_point_file`'s general meaning I think it's better to specifically say "which is passed as <argument name> to the Node.js resolution algorithm <link>".